### PR TITLE
Bittrex Ticker API support

### PR DESCRIPTION
**Pull request by luissantos**
I would like to add support for Bittrex exchange.
Please let me know if something else is required before I can submit my changes.

### CODE REVIEW

File: src/ApiService.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ import * as ProviderPaymium from './providers/ProviderPaymium';
 import * as ProviderPoloniex from './providers/ProviderBitPay';
 import * as ProviderSatangPro from './providers/ProviderSatangPro';
 import * as ProviderVccExchange from './providers/ProviderVccExchange';
+import * as ProviderBittrex from './providers/ProviderBittrex';
```
Comment (OttoAllmendinger) [inline src/ApiService.ts:29]: please add it in alphabetical order

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]
see comment

File: src/ApiService.ts (hunk 1 above)
Comment (luissantos) [reply src/ApiService.ts]: done.

**Comment by OttoAllmendinger** [comment]
thanks! merged

**Pull request closed without merging**